packages/next-source-maps/index.js

```diff
@@ -29,11 +29,11 @@ module.exports = (options = {}) => {
     })
   }
 
-  if (options.devtool) {
+  if (!pluginOptions.webpack) {
```

Comment:
hmm I think this would potentially fail if users to this:

```
const sourcemaps = require('@zeit/next-sourcemaps')

module.exports = sourcemaps()
```

and also when 

```
const sourcemaps = require('@zeit/next-sourcemaps')

module.exports = sourcemaps({})
```